### PR TITLE
Hiding Problem Dificulty fix

### PR DESCRIPTION
**Pull request by ykr5184**
Apparently nobody noticed that the view button was still the colour of the difficulty even when the difficulty tag was hidden. Oops. Fixed now! Let me know if you think a different colour should be used, but I think purple looks good and is difficulty ambiguous.

### CODE REVIEW

**Comment by ykr5184** [comment]
@myst6 just saw your (I think it was you) comment on the people page. My discord is the same as my GitHub name??? But don't add it to the website please.

File: content/people/instances/index.tsx [hunk 1]
```diff
@@ -62,13 +62,12 @@ export const people: Person[] = [
...
     name: "Yuvan Raja",
     socials: [
-      // i don't know yuvan's discord :C
       {
         platform: platforms.github,
         handle: "ykr5184",
       }
     ],
-    roles: ["Contributor"],
+    roles: ["Developer", "Contributor"],
```
Comment (myst-6) [inline content/people/instances/index.tsx:70]: I think it's fair.

File: components/Problem/index.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ Problem display can't be too long (>${MAXLEN}ch). \
             <Flex flex={1} justifyContent="flex-end">
               <Button 
                 size="sm" 
-                colorScheme={difficultyColors[problem.difficulty]} 
+                colorScheme={show.showDiff ? difficultyColors[problem.difficulty] : "purple"}
```
Comment (myst-6) [inline components/Problem/index.tsx:36]: Maybe I would prefer to use a colour that isn't associated with a difficulty already (purple is associated with insane).

However, I don't mind this choice, so I'll approve this PR. It's up to you if you want to change this colour.

**Review by myst-6 — APPROVED** [review]

**Comment by myst-6** [comment]
> @myst6 just saw your (I think it was you) comment on the people page. My discord is the same as my GitHub name??? But don't add it to the website please.

I actually meant to write 'codeforces handle', not discord username, lol

**Comment by myst-6** [comment]
Of course, you don't have to put your CF on there either... but just wanted to clarify that.

File: components/Problem/index.tsx (hunk 2 above)
Comment (macaquedev) [reply components/Problem/index.tsx:36]: Please change to grey.

**Review by macaquedev — CHANGES_REQUESTED** [review]
colour cannot be purple, please make it grey or something

**Comment by ykr5184** [comment]
> > @myst6 just saw your (I think it was you) comment on the people page. My discord is the same as my GitHub name??? But don't add it to the website please.
> 
> I actually meant to write 'codeforces handle', not discord username, lol. I won't put anyone's discords on there without asking, but I did want to put your CF on there.

Oh that makes more sense. I actually only made a CF account very recently so once I climb to a decent stable rating I'll add it 😅

**Comment by macaquedev** [comment]
do you want to tell us your CF on the server?

**Review by macaquedev — APPROVED** [review]
good work!

**Pull request merged**